### PR TITLE
Stop flowing Microsoft.NETCore.App

### PR DESCRIPTION
**Pull request by bricelam**
Having this seems to cripple the experience of running and debugging tests in Visual Studio.

### CODE REVIEW

**Review by ajcvickers — APPROVED** [review]
:-(

**Review by smitpatel — APPROVED** [review]

**Comment by bricelam** [comment]
It looks like Logging requires a runtime newer than Preview 4...

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -33,31 +34,15 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>1d2aa13ffc256a8c3f645de9c4d0f1d4f3296711</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Logging" Version="6.0.0-preview.6.21307.1">
+    <!-- NB: Pinned for compatibility with latest preview release -->
+    <Dependency Name="Microsoft.Extensions.Logging" Version="6.0.0-preview.4.21253.7" Pinned="True">
```
Comment (bricelam) [inline eng/Version.Details.xml:38]: 😬 That's going to put us on the "out-of-date dependencies" report...

Comment (smitpatel) [reply eng/Version.Details.xml:38]: Eventual consistency.

Comment (bricelam) [reply eng/Version.Details.xml]: Found the newest version that should work. It's only two weeks old.

Comment (bricelam) [reply eng/Version.Details.xml]: Now we'll get a PR from the bot that'll remain blocked until Preview 6 ships 😈

**Pull request merged**